spec/features/peeping_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'peeping'
+
+describe Tweet do 
+
+  it 'it views all peeps' do 
+		expect(Tweet.all.class). to eq Array
```

Comment:
This is great start. 
Perhaps one way of checking the `all` method could be to insert a row into the database at the beginning of this test and then check that the method returns an array of a Tweet with the same attributes as in the database?